modules/log6x-loki-sizing.adoc

```diff
@@ -0,0 +1,93 @@
+// Module is included in the following assemblies:
+// * observability/logging/logging-6.0/
+
+:_mod-docs-content-type: CONCEPT
+[id="log6x-loki-sizing_{context}"]
+= Loki deployment sizing
+
+Sizing for Loki follows the format of `1x.<size>` where the value `1x` is number of instances and `<size>` specifies performance capabilities.
```

Comment:
```suggestion
Sizing for Loki follows the `1x.<size>`format, where `1x` is number of instances and `<size>` specifies performance capabilities.
```

Reply:
The current wording has been in the docs for awhile, I see no reason to make this instance variant.